scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -1022,13 +1029,13 @@ TaskInfoImpl createTaskInfo(JobIdImpl jobId) {
         taskInfo.setExecutionDuration(getExecutionDuration());
         return taskInfo;
     }
-    
+
     public TaskInfo toTaskInfo() {
         JobIdImpl jobId = new JobIdImpl(getJobData().getId(), getJobData().getJobName());
         TaskInfoImpl taskInfo = createTaskInfo(jobId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "taskInfo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)